tests/features/search/search_in_data_entity.spec.ts

```diff
@@ -0,0 +1,252 @@
+  /**
+   * /project/1/test-cases/112
+   */
+  test(`Display an expected item with expression in tag`, async ({ steps: { pages } }) => {
+    await test.step(`When filling a valid expression in the search input`, async () => {
+      await pages.catalog.searchBy('tagForSearching');
+      expect(await pages.catalog.isListItemVisible(entityNameWithAlphabeticChars)).toBeTruthy();
+    });
+  });
+  /**
+   * /project/1/test-cases/113
+   */
+  test(`Display an expected item with expression in custom metadata`, async ({
+    steps: { pages },
+  }) => {
+    await test.step(`When filling a valid expression in the search input`, async () => {
+      await pages.catalog.searchBy(entityNameWithAlphabeticChars);
+      await pages.catalog.clickOnListItem(entityNameWithAlphabeticChars);
+      await pages.overview.createCustomMetadata('isOnSale');
+      await pages.topPanel.clickTab('Catalog');
+      await pages.catalog.searchBy('isOnSale');
+      expect(await pages.catalog.isListItemVisible(entityNameWithAlphabeticChars)).toBeTruthy();
+    });
+  });
+  /**
+   * /project/1/test-cases/114
+   */
+  test(`Display an expected item with expression in dataset field name`, async ({
+    steps: { pages },
+  }) => {
+    await test.step(`When filling a valid expression in the search input`, async () => {
+      await pages.catalog.searchBy('genre_id');
+      expect(await pages.catalog.isListItemVisible(entityWithStructure)).toBeTruthy();
+    });
+  });
+  /**
+   * /project/1/test-cases/115
+   */
+  test(`Display an expected item with expression in namespace`, async ({ steps: { pages } }) => {
+    await test.step(`When filling a valid expression in the search input`, async () => {
+      await pages.catalog.searchBy('ETL');
+      expect(await pages.catalog.isListItemVisible(entityNameWithAlphabeticChars)).toBeTruthy();
+    });
+  });
+  /**
+   * /project/1/test-cases/116
+   */
+  test(`Display an expected item with expression in datasource name`, async ({
+    steps: { pages },
+  }) => {
+    await test.step(`When filling a valid expression in the search input`, async () => {
+      await pages.catalog.searchBy('oracle_airplanes');
+      expect(await pages.catalog.isListItemVisible(entityNameWithNumbers)).toBeTruthy();
+    });
+  });
+  /**
+   * /project/1/test-cases/118
+   */
+  test(`Display an expected item with expression in owner's name`, async ({ steps: { pages } }) => {
+    const ownerName4 = 'ownerName4';
+    const ownerTitle4 = 'ownerTitle4';
+    await test.step(`When filling a valid expression in the search input`, async () => {
+      await pages.catalog.searchBy(entityNameWithAlphabeticChars);
+      await pages.catalog.clickOnListItem(entityNameWithAlphabeticChars);
+      await pages.overview.createOwner(ownerName4, ownerTitle4);
+      await pages.topPanel.clickTab('Catalog');
+      await pages.catalog.searchBy(ownerName4);
+      expect(await pages.catalog.isListItemVisible(entityNameWithAlphabeticChars)).toBeTruthy();
```

Comment:
These test lines are appearing in several tests below, try to separate them into one group and create parametrized one test for them all https://playwright.dev/docs/test-parameterize